packages/extension/package.json

```diff
@@ -15,6 +15,11 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "simple-cookie": "^1.0.15",
-    "use-context-selector": "^1.4.1"
+    "tldts": "^6.0.8",
+    "use-context-selector": "^1.4.1", 
+    "victory": "^36.6.11"
+  },
+  "devDependencies":{
+    "@tailwindcss/line-clamp": "^0.4.4"
```

Comment:
line-clamp plugin has been integrated to core tailwind package. This can be removed now.